docs/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ set (PORTAL_GEN_XSL ${CMAKE_CURRENT_BINARY_DIR}/BuildTools/PortalGen.xsl)
 
 if ( "${HPCC_MATURITY}" STREQUAL "release" )
 	set (DOC_VERSION "${DOC_VERSION}-${HPCC_SEQUENCE}")
+	set (DOC_VERSION_ONLY "${DOC_VERSION}")
```

Comment:
Feels like it's fixing the symptom rather than the cause.

Reply:
The cause was a change in the docMacros.  The old way of titling documents would have the major.minor.point-sequence variables fill in the end of the document name.  Now that's been replaced by 'DOC_VERSION_ONLY'. This was part of a ticket completed by Ming so that the document builds could have git tags in them.